dask/diagnostics/profile_visualize.py

```diff
@@ -146,7 +146,7 @@ def get_colors(palette, funcs):
     return [color_lookup[n] for n in funcs]
 
 
-def visualize(profilers, file_path=None, show=True, save=True, **kwargs):
+def visualize(profilers, file_path=None, show=True, save=True, mode=None, **kwargs):
```

Comment:
I think this change is already in a different PR.

Reply:
@jsignell 
yes, you are right.
(This is my first steps in github. I presume I should have created a new branch for this last PR to avoid mismatching with previous PR.)
Concerning your previous remark (`iterable of strings` instead of `list of strings`), I agree with you.
I let you finalize the merge and close this PR.
Patrick